android/NavigationExamples/app/src/main/java/com/trimblemaps/navigationexamples/NavigationActivity.kt

```diff
@@ -152,6 +152,10 @@ class NavigationActivity : AppCompatActivity() {
         override fun onFinalDestinationArrival(routeProgress: RouteProgress) {
         }
 
+        override fun onIntermediateArrival(routeLegProgress: RouteLegProgress) {
+            TODO("Not yet implemented")
```

Comment:
This TODO would be treated as an exception and I believe look like a crash. Let's just leave the body empty like the others.